src/index.test.ts

```diff
@@ -2268,3 +2289,54 @@ describe('Experimental options togglesStorageTTL enabled', () => {
         });
     });
 });
+
+describe('updateToggles', () => {
```

Comment:
I wanted to suggest that we don't add the word `toggles` in the public API since we already migrated most UI facing code to `flags` instead of `toggles`. But since we already have a public method called `getAllToggles` in this SDK then introducing another convention adds more confusion rather than improving clarity.

Reply:
So no action needed IMO